### ISSUE TITLE
Use AJV plugins with ajv-cli?

### ISSUE THREAD

**Issue by clenk** [ajv-validator/ajv-cli#80]
Is there a way to specify AJV plugins for ajv-cli to use?

Specifically, I want to use https://github.com/luzlab/ajv-formats-draft2019 when running `ajv compile` to avoid errors about `unknown format "idn-hostname" is used in schema at path...`. Thanks.

**Comment by epoberezkin** [comment]
Yes, see -c option: https://github.com/ajv-validator/ajv-cli#-c---custom-keywordsformats-definitions

Probably readme needs to clarify that any plugin can be passed this way.

**Comment by danez** [comment]
I have the same problem, after updating to 4.0 formats are throwing an error:
> error: unknown format "uuid" ignored in schema at path "#/items/properties/uuid"

Shouldn't the `ajv-formats` plugin be enabled by default?

**Comment by anapaulagomes** [comment]
In case someone is wondering, this is how I managed to do it:

```
npm install -g ajv-cli ajv-formats
ajv compile --spec draft2020 -c ajv-formats -s schemas/v1/schema.json 
```
